Sources/KlaviyoSwiftExtension/KlaviyoExtension.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  KlaviyoExtension.swift
+//
+//
+//  Created by Ajay Subramanya on 6/23/23.
+//
+#if !os(macOS)
+import Foundation
```

Comment:
foundation is not on macos?

Reply:
when running the package tests using xcodebuild on CLI there wasn't a way to indicate to run only on iOS and this was a suggestion I found online. I did notice that the package.swift file did have the platform in there but still I was getting compile error about something in here not supported. Checking now to see what happens when I remove this.

Reply:
removed this now. looks like somehow the package defaults and build for a mac target even though the package clearly has the platform as iOS.